DependencyInjection/Configuration.php

```diff
@@ -17,8 +17,13 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('massive_art_pdf');
+        $treeBuilder = new TreeBuilder('massive_art_pdf');
```

Comment:
I assume this should be `massive_pdf` instead of `massive_art_pdf`

Reply:
Configuring this bundle using `massive_art_pdf` leads to the following error: `There is no extension able to load the configuration for "massive_art_pdf"`, therefore I changed it to `massive_pdf`